FAIL
Kernel panic in run: PANIC at ../../threads/thread.c:679 in schedule(): assertion `is_thread (next)' failed.
Call stack: 0x80042187ad 0x8004207e72 0x8004206ef0 0x800420a314 0x800421bfa3 0x800421de2d 0x800421d9cb 0x800421d840 0x400106 0x4001b3 0x400c54
Translation of call stack:
0x00000080042187ad: debug_panic (lib/kernel/debug.c:32)
0x0000008004207e72: schedule (threads/thread.c:681)
0x0000008004206ef0: thread_block (threads/thread.c:241)
0x000000800420a314: sema_down (threads/synch.c:68)
0x000000800421bfa3: process_fork (userprog/process.c:115)
0x000000800421de2d: fork (userprog/syscall.c:179)
0x000000800421d9cb: syscall_handler (userprog/syscall.c:85)
0x000000800421d840: no_sti (userprog/syscall-entry.o:?)
0x0000000000400106: (unknown)
0x00000000004001b3: (unknown)
0x0000000000400c54: (unknown)
